ibm/service/kms/resource_ibm_kms_key_alias_test.go

```diff
@@ -111,6 +111,15 @@ func TestAccIBMKMSResource_Key_Alias_Key_Check(t *testing.T) {
 					resource.TestCheckResourceAttr("ibm_kms_key_alias.testAlias", "alias", aliasName2),
 				),
 			},
+			{
+				Config: testAccCheckIBMKmsResourceAliasWithExistingAlias(instanceName, keyName, aliasName, aliasName2),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("ibm_kms_key.test", "key_name", keyName),
+					resource.TestCheckResourceAttr("ibm_kms_key_alias.testAlias", "alias", aliasName),
+					resource.TestCheckResourceAttr("ibm_kms_key_alias.testAlias2", "existing_alias", aliasName),
+					resource.TestCheckResourceAttr("ibm_kms_key_alias.testAlias2", "alias", aliasName2),
```

Comment:
the attribute name `ibm_kms_key_alias.testAlias2` is repeating

Reply:
The two attributes are testing two different variables `existing_alias` and `alias`